roles/lnls-ans-role-repositories/vars/Debian-buster.yml

```diff
@@ -1,4 +1,7 @@
 ---
+# Proxy APT
+proxy_address: http://10.0.38.42:3142
```

Comment:
@rbatistacardoso , is there a particular reason for this variable to be defined in every debian distro? Could it be a global variable?

Reply:
I believe that the variable `proxy_address` should be defined at [sirius/group_vars/control_room_all.yml](https://github.com/lnls-sirius/lnls-ansible/blob/master/inventories/sirius/group_vars/control_room_all.yml) and the default definitions removed from the role.

Also, it would be nice if the role performed a check to see if the variable is defined or not and take actions accordingly.
```yml
  - debug:
      msg: "foo is defined and foo == '' and foo != None"
    when: foo is defined and foo == '' and foo != None
```
Maybe guarantee that the file `/etc/apt/apt.conf.d/proxy` is removed when the variable is undefined or empty,